### PR TITLE
Fix ILMM Docstring

### PR DESCRIPTION
**Pull request by willtebbutt**
Looks like it got out of date.

### CODE REVIEW

File: src/ilmm.jl [hunk 1]
```diff
@@ -1,19 +1,16 @@
 """
-    ILMM(fs, H)
+    ILMM(f, H)
+
 An Instantaneous Linear Mixing Model (ILMM) -- a distribution over vector-
 valued functions. Let `p` be the number of observed outputs, and `m` the number of latent
 processes, then `H`, also known as the mixing matrix, is a `p x m` matrix whose column
-space spans the output space. The latent processes are represented a Gaussian process `f`.
+space spans the output space. `H` "mixes" a latent multi-output AbstractGP `f`.
```
Comment (thomasgudjonwright) [inline src/ilmm.jl:7]: Is "mixes" maybe a bit too ambiguous or vague? Or given we have explained what `H` is are we just introducing the term "mixes"

**Review by thomasgudjonwright — APPROVED** [review]
Just one comment, but LGTM!

File: src/ilmm.jl (hunk 1 above)
Comment (willtebbutt) [reply src/ilmm.jl:7]: I was referencing the fact that call it the mixing matrix above, but I agree we should make it more precise.

How about something like
 ```suggestion
space spans the output space. Given a sample from `f`, the value of the ILMM at any input `x` for all outputs is
```julia
H * [f((x, 1)), f((x, 2)), ..., f((x, m))]
```
```

Comment (thomasgudjonwright) [reply src/ilmm.jl:7]: That makes it super clear! Once that is commited I am happy with this being merged!

**Comment by willtebbutt** [comment]
Thanks for the review. I'll merge once CI passes.

File: src/ilmm.jl [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-    ILMM(fs, H)
+    ILMM(f, H)
+
 An Instantaneous Linear Mixing Model (ILMM) -- a distribution over vector-
 valued functions. Let `p` be the number of observed outputs, and `m` the number of latent
 processes, then `H`, also known as the mixing matrix, is a `p x m` matrix whose column
-space spans the output space. The latent processes are represented a Gaussian process `f`.
+space spans the output space. Given a sample from `f`, the value of the ILMM at any input `x` for all outputs is
+```julia
+H * [f((x, 1)), f((x, 2)), ..., f((x, m))]
```
Comment (thomasgudjonwright) [inline src/ilmm.jl:10]: It won't let me make this a suggestion given the suggestion uses the "```" but I think we need to close the julia code block!

File: src/ilmm.jl [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
```
Comment (willtebbutt) [inline src/ilmm.jl:9]:
```suggestion
```julia
H * [f((x, 1)), f((x, 2)), ..., f((x, m))]
```
```

File: src/ilmm.jl (hunk 2 above)
Comment (willtebbutt) [reply src/ilmm.jl:10]: Oh, weird. I'll have to modify locally -- the suggestion API doesn't seem to like it.

**Pull request merged**